client/src/Movies/Movie.js

```diff
@@ -11,7 +11,7 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id = `${this.props.match.params.id}`;
```

Comment:
Why is this wrapped in a template literal string? 
Why not just do:
```javascript
const id = this.props.match.params.id;
```
or better yet:
```javascript
const { id } = this.props.match.params;
```